README.adoc

```diff
@@ -1,86 +1,132 @@
-= devops-stack-module-template
+= devops-stack-module-velero
 // Document attributes to replace along the document
 // Here you can define variables for something that keeps repeating along the text
 :chart-version: 5.0.2
```

Comment:
```suggestion
:velero-chart-version: 5.0.2
```

Reply:
You need to change this attribute, so the automatic updater also changes the attribute on the README.adoc. The prefix of the attribute is the same name as the name of the dependency on the Chart.yaml.